File: pmh/intellij/src/aa/bb/cc/AA.java
package aa.bb.cc;
// 오버로딩
public class AA {
    public void doA(){
        System.out.println("기본 doA 메서드");
    }
    public void doA(int a){
        System.out.println("int a 있는 doA 메서드");
    }
    public void doA(double a){
        System.out.println("double a 있는 doA 메서드");
    }
}

File: myreact/ex0321/src/components/AA.js
import React, { useState } from 'react';

export default function AA(){
    const [aa,setAa] = useState('aaa');
    function doA(){
        aa === "aaa"? setAa("bbb"):setAa("aaa");
    }
    return(
        <div>
            <h1>{aa}</h1>
            <button onClick={doA}>값바꾸기</button>
        </div>
    )
}